packages/dx-react-demos/src/material-ui/filtering/custom-filter-row.jsx

```diff
@@ -31,26 +33,45 @@ const styles = theme => ({
     width: '100%',
     paddingLeft: theme.spacing.unit,
     paddingRight: theme.spacing.unit,
+    paddingTop: theme.spacing.unit * 1.5,
+    verticalAlign: 'top',
+  },
+  placeholder: {
+    color: 'rgba(0,0,0,0.36)',
```

Comment:
Use color from the [theme](https://material-ui-1dab0.firebaseapp.com/customization/themes/#the-other-variables) object.